### DISCUSSION TITLE
Is there a plan for the release of minor updates for 0.10 series?

### DISCUSSION THREAD

**Discussion by vvihar** [django-wiki/django-wiki#1349] [Q&A] [answered]
Hi, I am using wiki 0.10 on Django 4.2.11.

I am now trying to create a markdown plugin to enable a strikethrough expression (~~text~~).

However, due to issue #1314, I am not successful in achieving my goal.
This is because the bug itself has already been fixed, but there have been no releases which include this bug fix so far.

I thought of directly installing the latest version from GitHub, but it is not desirable for me judging from our organization's policy.

I would like to ask a question whether there is a plan for releasing the minor updates for 0.10.
If there is, when would it be?

Thank you!

**Comment by benjaoming** [comment]
Hi!

There will be both a minor release and a 0.11. Sorry, I haven't had time to do this recently but it's on my soon-TODO :)

**Reply by vvihar** [reply]
Understood, thank you for a reply!

**Reply by benjaoming** [reply]
Just noticed this:

> I thought of directly installing the latest version from GitHub, but it is not desirable for me judging from our organization's policy.

I agree with that policy. I think, as a maintainer, I would also not anticipate or want to support that scenario.

**Comment by oscarmcm** [comment]
@benjaoming I can create a new release.

**Reply by benjaoming** [reply]
@oscarmcm the current `main` branch is targeted for the next major release, and we have a security patch in between for a patch release... we decided to do 0.10.1 before 0.11 (and this one should include #1337 etc)

**Reply by oscarmcm** [reply]
the security patch is done? or do we have something there missing?

**Comment by benjaoming** [comment, marked as answer]
Hi @vvihar 0.11 is out with support for new Django versions etc: https://django-wiki.readthedocs.io/en/main/release_notes.html
